test-programs/sdk-test/src/lib.rs

```diff
@@ -0,0 +1,296 @@
+use std::{
+    io::Cursor,
+    mem,
+    str::{self, Utf8Error},
+};
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use light_sdk::{
+    account_info::LightAccountInfo,
+    account_meta::LightAccountMeta,
+    address::derive_address,
+    error::LightSdkError,
+    hasher::Discriminator,
+    instruction_data::LightInstructionData,
+    program_merkle_context::unpack_address_merkle_context,
+    proof::ProofRpcResult,
+    verify::{verify_light_accounts, LightCpiAccounts},
+    LightDiscriminator, LightHasher,
+};
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    entrypoint::{self, ProgramResult},
+    program_error::ProgramError,
+    pubkey::Pubkey,
+};
+use thiserror::Error;
+
+entrypoint!(process_instruction);
```

Comment:
file would be easier to read if we put the implementation of process_instruction up here as well.